### ISSUE TITLE
Some Bugs

### ISSUE THREAD

**Issue by techmastersofficial** [Constantine-k/brackets-psd-layer-viewer#1]
- Export Png Btn Not Wrking 
- Export Png Dropdown Not Wrking
- I downloaded The Image from PSD From Top Bar [Canvas] But Did'nt Find Where File Is Save Please add A Save Option So User can Locate Where To Save Image 
- Add Resizing Options To Side Bars On Left And Rights So Users can Adjusts The PSD preview .

**Comment by Constantine-k** [comment]
> Export Png Btn Not Wrking
> Export Png Dropdown Not Wrking
> I downloaded The Image from PSD From Top Bar [Canvas] But Did'nt Find Where File Is Save Please add A Save Option So User can Locate Where To Save Image

Yeah, unfortunately, as I mentioned in description, Export PNG button doesn't work. So to save image you should just drag the image above to your desktop or needed folder.

> Add Resizing Options To Side Bars On Left And Rights So Users can Adjusts The PSD preview .

There is a resizing tool at the right-bottom corner (it shows after opening psd file).


The main issue with this extension is that it's just an iframe to site http://psd.keyangxiang.com/. Because of that I can't really change too much. I got a permission from the site's author to make extension with an iframe (so if there's an ad on the site, it will be shown in extension as well), but I'm not sure about making fully local extension with his code.

**Comment by techmastersofficial** [comment]
ok thanks for solution and quick reply :)

**Comment by Constantine-k** [comment]
You are welcome :)
